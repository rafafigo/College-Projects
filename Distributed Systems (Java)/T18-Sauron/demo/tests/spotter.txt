init ../demo/tests/data/init.txt
spot car 00NX08
spot car 0*
spot car *
spot car *12*
spot car *1*A*
spot car *1*AA*
spot car 0*0*
spot person 1012435145270653984
spot person 1*
spot person *
spot person *12
spot person 1*2
spot person 1*2*3
spot person 90*
trail person 1012435145270653984
trail car 00NX08
trail person 1
clear
