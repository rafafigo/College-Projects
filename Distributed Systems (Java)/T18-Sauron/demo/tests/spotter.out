SpotterApp
Received 2 arguments
arg[0] = localhost
arg[1] = 8080
CAR,00NX08,2008-07-18T12:38:04,_Camera_29,56.088966,-65.741688
CAR,00NX08,2008-07-18T12:38:04,_Camera_29,56.088966,-65.741688
CAR,00UI25,1998-05-03T00:27:49,_Camera_43,45.588239,-82.034685
CAR,02LE43,1967-04-25T16:48:53,_Camera_34,-76.036835,88.093573
CAR,04WK88,1968-08-17T04:28:27,_Camera_42,8.704303,134.719516
CAR,05FD60,2001-11-22T02:27:37,_Camera_37,27.929274,119.050144
CAR,05HM64,2004-10-23T23:46:20,_Camera_27,-48.985696,-45.254164
CAR,05KQ92,1989-07-25T07:59,_Camera_6,70.834569,176.96735
CAR,05QM01,1964-09-20T09:59:17,_Camera_6,70.834569,176.96735
CAR,05WT25,1995-08-21T18:17:30,_Camera_27,-48.985696,-45.254164
CAR,07DV99,1953-01-27T08:21:45,_Camera_36,-12.993477,174.878086
CAR,07EO15,1996-03-28T13:44:55,_Camera_39,-9.213224,19.584055
CAR,08XC38,2004-06-23T11:17:45,_Camera_5,1.967547,49.305173
CAR,09VW87,2003-03-09T11:53:53,_Camera_27,-48.985696,-45.254164
CAR,00NX08,2008-07-18T12:38:04,_Camera_29,56.088966,-65.741688
CAR,00UI25,1998-05-03T00:27:49,_Camera_43,45.588239,-82.034685
CAR,02LE43,1967-04-25T16:48:53,_Camera_34,-76.036835,88.093573
CAR,04WK88,1968-08-17T04:28:27,_Camera_42,8.704303,134.719516
CAR,05FD60,2001-11-22T02:27:37,_Camera_37,27.929274,119.050144
CAR,05HM64,2004-10-23T23:46:20,_Camera_27,-48.985696,-45.254164
CAR,05KQ92,1989-07-25T07:59,_Camera_6,70.834569,176.96735
CAR,05QM01,1964-09-20T09:59:17,_Camera_6,70.834569,176.96735
CAR,05WT25,1995-08-21T18:17:30,_Camera_27,-48.985696,-45.254164
CAR,07DV99,1953-01-27T08:21:45,_Camera_36,-12.993477,174.878086
CAR,07EO15,1996-03-28T13:44:55,_Camera_39,-9.213224,19.584055
CAR,08XC38,2004-06-23T11:17:45,_Camera_5,1.967547,49.305173
CAR,09VW87,2003-03-09T11:53:53,_Camera_27,-48.985696,-45.254164
CAR,10WC43,1981-01-23T23:18:48,_Camera_37,27.929274,119.050144
CAR,11AI60,1957-12-17T12:42:43,_Camera_21,39.787559,-140.379141
CAR,11GD01,1994-03-21T11:43:34,_Camera_50,-62.14971,-67.853099
CAR,12DM67,2014-12-09T14:35:45,_Camera_45,-36.849172,39.740283
CAR,15CK66,1970-08-06T06:49,_Camera_36,-12.993477,174.878086
CAR,15GQ96,2009-11-01T09:01:42,_Camera_11,-28.87645,-99.168322
CAR,15HI90,1954-07-29T17:38:27,_Camera_25,74.794776,154.474923
CAR,16AL25,1940-07-03T14:57:50,_Camera_5,1.967547,49.305173
CAR,19MZ03,2006-02-15T18:55:56,_Camera_42,8.704303,134.719516
CAR,22GW92,1944-06-10T08:28:15,_Camera_27,-48.985696,-45.254164
CAR,25CZ93,2005-03-15T20:49:38,_Camera_19,-76.539366,-121.880941
CAR,25IV94,1958-12-06T16:55:25,_Camera_36,-12.993477,174.878086
CAR,28QN62,2014-04-27T09:34:12,_Camera_46,-70.676761,-26.562017
CAR,29JZ61,1946-12-07T08:53:10,_Camera_16,-40.039633,-151.282119
CAR,30HH23,1996-07-08T19:45:33,_Camera_10,-77.559168,-94.815517
CAR,32KB01,1997-04-17T22:48:56,_Camera_40,-11.619677,-68.571013
CAR,32LM94,2016-03-22T19:06:40,_Camera_37,27.929274,119.050144
CAR,34EK34,2016-07-06T00:44:41,_Camera_21,39.787559,-140.379141
CAR,35DF79,1958-12-10T01:06:06,_Camera_2,55.170086,-163.81953
CAR,36FB52,2016-12-31T04:45:09,_Camera_25,74.794776,154.474923
CAR,36ZL59,1955-09-03T09:33:08,_Camera_6,70.834569,176.96735
CAR,37QP76,1977-05-06T06:42:05,_Camera_27,-48.985696,-45.254164
CAR,39BW33,1936-05-09T00:26:26,_Camera_11,-28.87645,-99.168322
CAR,39GO99,2002-08-18T09:11:33,_Camera_49,-32.003907,-0.817517
CAR,39OU65,1932-02-17T22:10:35,_Camera_39,-9.213224,19.584055
CAR,39ZS61,1968-05-09T07:11:51,_Camera_38,-15.906281,-7.659154
CAR,41IO60,2001-08-15T10:41:22,_Camera_38,-15.906281,-7.659154
CAR,41VJ82,1977-05-24T02:13:28,_Camera_27,-48.985696,-45.254164
CAR,43AV94,1980-06-02T03:45:29,_Camera_36,-12.993477,174.878086
CAR,43FW89,2002-02-28T11:59:56,_Camera_46,-70.676761,-26.562017
CAR,43TV03,1996-03-30T14:16:51,_Camera_20,40.331394,-135.398244
CAR,43UW51,2007-08-02T04:13:11,_Camera_13,3.392588,131.009408
CAR,45MH19,1986-05-09T10:23:26,_Camera_24,-64.407795,133.612736
CAR,45ZY14,1945-02-25T21:01:37,_Camera_13,3.392588,131.009408
CAR,50BO83,1946-01-16T11:54:27,_Camera_25,74.794776,154.474923
CAR,50LI03,1944-06-01T03:56:37,_Camera_46,-70.676761,-26.562017
CAR,51AT02,1963-06-02T22:27:25,_Camera_25,74.794776,154.474923
CAR,52OD64,1965-02-11T09:21:31,_Camera_48,-12.282729,-150.787278
CAR,53EC30,2009-05-10T09:32:02,_Camera_31,66.674969,-114.307091
CAR,55OI03,1943-08-24T09:19:37,_Camera_29,56.088966,-65.741688
CAR,55XR67,2018-11-17T09:42:45,_Camera_38,-15.906281,-7.659154
CAR,57EB34,1961-10-15T10:44:26,_Camera_31,66.674969,-114.307091
CAR,57IT28,1966-03-17T18:46:51,_Camera_6,70.834569,176.96735
CAR,57XT82,1953-02-16T11:47:11,_Camera_31,66.674969,-114.307091
CAR,58LH79,1987-04-20T10:11:21,_Camera_49,-32.003907,-0.817517
CAR,59KM43,1969-04-07T04:24:42,_Camera_38,-15.906281,-7.659154
CAR,59MQ89,1939-01-16T18:38:24,_Camera_15,-42.664172,70.18845
CAR,61VY38,2008-01-30T10:08:01,_Camera_40,-11.619677,-68.571013
CAR,63FM58,1953-01-27T08:21:45,_Camera_19,-76.539366,-121.880941
CAR,63PB81,1932-08-30T23:12:25,_Camera_8,-57.389193,75.017573
CAR,63RL54,2006-02-15T18:55:56,_Camera_31,66.674969,-114.307091
CAR,63VV25,1938-01-26T12:52:36,_Camera_5,1.967547,49.305173
CAR,64OD47,1967-11-09T22:42:30,_Camera_15,-42.664172,70.18845
CAR,64UQ40,1984-11-25T15:10:35,_Camera_10,-77.559168,-94.815517
CAR,65TJ97,1959-05-23T04:35:59,_Camera_13,3.392588,131.009408
CAR,65UV44,2003-04-13T04:51:21,_Camera_37,27.929274,119.050144
CAR,66CH68,1952-06-18T08:23:26,_Camera_29,56.088966,-65.741688
CAR,66OZ49,1979-06-19T02:39:12,_Camera_11,-28.87645,-99.168322
CAR,67FJ21,1939-12-30T04:12:46,_Camera_40,-11.619677,-68.571013
CAR,69MG57,1940-07-17T08:02:41,_Camera_50,-62.14971,-67.853099
CAR,69XN35,1968-05-09T02:59:12,_Camera_43,45.588239,-82.034685
CAR,72SK39,2006-10-27T14:27:42,_Camera_46,-70.676761,-26.562017
CAR,73AZ17,1987-11-30T17:17:11,_Camera_32,41.717836,28.079691
CAR,73SX84,1995-05-11T05:55:07,_Camera_13,3.392588,131.009408
CAR,73TP73,1930-03-12T20:09:53,_Camera_15,-42.664172,70.18845
CAR,73WQ30,1930-07-16T13:08:41,_Camera_43,45.588239,-82.034685
CAR,76VX48,1954-06-14T13:17:08,_Camera_31,66.674969,-114.307091
CAR,80IF31,1967-02-13T20:13:41,_Camera_11,-28.87645,-99.168322
CAR,81JO13,1947-10-20T14:25:23,_Camera_27,-48.985696,-45.254164
CAR,81OJ43,1933-09-27T01:56:01,_Camera_41,16.696536,89.621778
CAR,82KT11,1950-11-28T00:49:27,_Camera_40,-11.619677,-68.571013
CAR,82RS27,1946-06-10T14:06:09,_Camera_43,45.588239,-82.034685
CAR,83RN07,2019-11-23T15:58:41,_Camera_20,40.331394,-135.398244
CAR,84OF91,2010-04-09T18:51:28,_Camera_23,59.327018,-106.052814
CAR,85VB14,1943-08-21T06:38:27,_Camera_11,-28.87645,-99.168322
CAR,86NY86,1931-12-07T03:12:25,_Camera_24,-64.407795,133.612736
CAR,86YV47,1932-09-21T15:51:09,_Camera_36,-12.993477,174.878086
CAR,90FT94,1976-06-10T22:32:15,_Camera_45,-36.849172,39.740283
CAR,90NT12,1944-06-27T11:40:43,_Camera_36,-12.993477,174.878086
CAR,90PE04,1953-10-21T22:42:21,_Camera_18,-80.110288,-50.820967
CAR,90ZJ75,1960-04-19T17:29:46,_Camera_16,-40.039633,-151.282119
CAR,91CQ92,1982-07-19T11:34:32,_Camera_40,-11.619677,-68.571013
CAR,91FE21,2015-08-15T10:19:30,_Camera_19,-76.539366,-121.880941
CAR,91LH39,1954-04-12T14:55:50,_Camera_42,8.704303,134.719516
CAR,91TM61,1942-05-03T20:40:04,_Camera_49,-32.003907,-0.817517
CAR,94BQ23,1939-06-29T18:42:35,_Camera_23,59.327018,-106.052814
CAR,94II30,1966-04-12T20:07:34,_Camera_43,45.588239,-82.034685
CAR,95VE10,2019-08-16T02:40:11,_Camera_50,-62.14971,-67.853099
CAR,97DF39,1975-09-04T08:47:22,_Camera_40,-11.619677,-68.571013
CAR,99TK64,1997-07-31T12:02:58,_Camera_40,-11.619677,-68.571013
CAR,12DM67,2014-12-09T14:35:45,_Camera_45,-36.849172,39.740283
CAR,90NT12,1944-06-27T11:40:43,_Camera_36,-12.993477,174.878086
CAR,11AI60,1957-12-17T12:42:43,_Camera_21,39.787559,-140.379141
CAR,16AL25,1940-07-03T14:57:50,_Camera_5,1.967547,49.305173
CAR,51AT02,1963-06-02T22:27:25,_Camera_25,74.794776,154.474923

CAR,00NX08,2008-07-18T12:38:04,_Camera_29,56.088966,-65.741688
CAR,00UI25,1998-05-03T00:27:49,_Camera_43,45.588239,-82.034685
CAR,05FD60,2001-11-22T02:27:37,_Camera_37,27.929274,119.050144
CAR,05QM01,1964-09-20T09:59:17,_Camera_6,70.834569,176.96735
PERSON,1012435145270653984,2011-05-27T15:20:04,_Camera_43,45.588239,-82.034685
PERSON,1012435145270653984,2011-05-27T15:20:04,_Camera_43,45.588239,-82.034685
PERSON,1199795135596102925,2007-03-21T12:51:10,_Camera_48,-12.282729,-150.787278
PERSON,1484037905483886915,2011-01-06T03:48:50,_Camera_31,66.674969,-114.307091
PERSON,1491264515069720082,1930-11-22T02:14:54,_Camera_7,38.991455,59.186553
PERSON,1634671514261703547,2010-09-25T13:21:51,_Camera_15,-42.664172,70.18845
PERSON,1834940460737294204,1979-09-22T00:46:15,_Camera_25,74.794776,154.474923
PERSON,1874472655587181950,1934-07-18T11:31:25,_Camera_24,-64.407795,133.612736
PERSON,190774122026117782,2012-04-30T00:49:50,_Camera_24,-64.407795,133.612736
PERSON,1012435145270653984,2011-05-27T15:20:04,_Camera_43,45.588239,-82.034685
PERSON,1199795135596102925,2007-03-21T12:51:10,_Camera_48,-12.282729,-150.787278
PERSON,1484037905483886915,2011-01-06T03:48:50,_Camera_31,66.674969,-114.307091
PERSON,1491264515069720082,1930-11-22T02:14:54,_Camera_7,38.991455,59.186553
PERSON,1634671514261703547,2010-09-25T13:21:51,_Camera_15,-42.664172,70.18845
PERSON,1834940460737294204,1979-09-22T00:46:15,_Camera_25,74.794776,154.474923
PERSON,1874472655587181950,1934-07-18T11:31:25,_Camera_24,-64.407795,133.612736
PERSON,190774122026117782,2012-04-30T00:49:50,_Camera_24,-64.407795,133.612736
PERSON,2105735261003063500,1986-01-12T07:29:29,_Camera_29,56.088966,-65.741688
PERSON,2142144219626739124,2005-03-15T20:49:38,_Camera_40,-11.619677,-68.571013
PERSON,2156207585215361201,2002-08-14T00:13:14,_Camera_45,-36.849172,39.740283
PERSON,2248301660987512276,1972-04-25T04:25:37,_Camera_49,-32.003907,-0.817517
PERSON,2324576691079921173,2006-08-20T09:40,_Camera_49,-32.003907,-0.817517
PERSON,2380207713246100832,1982-05-20T21:59:59,_Camera_21,39.787559,-140.379141
PERSON,2889678301911450650,1975-11-06T03:35:10,_Camera_24,-64.407795,133.612736
PERSON,312867692468293989,2012-10-02T01:16:36,_Camera_8,-57.389193,75.017573
PERSON,3256563700944043519,1950-04-29T23:01:08,_Camera_5,1.967547,49.305173
PERSON,3286791971098273593,1935-02-25T19:18:59,_Camera_36,-12.993477,174.878086
PERSON,3298928417441705478,2014-05-17T21:38:54,_Camera_2,55.170086,-163.81953
PERSON,3514267229424239705,1973-09-21T01:35:34,_Camera_25,74.794776,154.474923
PERSON,3535855477432604723,1953-07-10T14:29:32,_Camera_13,3.392588,131.009408
PERSON,3613445852177145686,2020-11-21T19:04:43,_Camera_37,27.929274,119.050144
PERSON,3785961708813391095,1935-08-06T01:35:45,_Camera_20,40.331394,-135.398244
PERSON,384841743166095423,1961-10-17T01:14:57,_Camera_16,-40.039633,-151.282119
PERSON,3884514239095838819,1985-09-16T17:28:43,_Camera_7,38.991455,59.186553
PERSON,3906176179227516677,1984-08-16T05:06:02,_Camera_13,3.392588,131.009408
PERSON,4046930077671164205,1974-03-30T09:46:02,_Camera_5,1.967547,49.305173
PERSON,409901112409802215,1981-09-21T10:55:56,_Camera_15,-42.664172,70.18845
PERSON,4365147080721328036,1936-04-04T09:22:50,_Camera_38,-15.906281,-7.659154
PERSON,4471433174805807791,1968-10-09T06:13:47,_Camera_6,70.834569,176.96735
PERSON,4652295885441839210,1971-02-27T06:00:05,_Camera_10,-77.559168,-94.815517
PERSON,4737741055082484811,1952-12-22T20:24:25,_Camera_21,39.787559,-140.379141
PERSON,4839111172016853318,1995-04-18T06:33:30,_Camera_23,59.327018,-106.052814
PERSON,4845546129355841690,2013-02-19T09:01:33,_Camera_11,-28.87645,-99.168322
PERSON,4847176387014693723,1946-05-03T11:09:47,_Camera_44,81.032238,-16.230442
PERSON,4871213061710265916,1953-06-17T13:12,_Camera_37,27.929274,119.050144
PERSON,4979367798045138151,1997-07-15T13:06:30,_Camera_8,-57.389193,75.017573
PERSON,5007040932783887788,2011-08-01T19:41:21,_Camera_49,-32.003907,-0.817517
PERSON,5133962056263303465,1980-02-02T03:08:57,_Camera_18,-80.110288,-50.820967
PERSON,5190212891189019117,2014-02-13T04:21:33,_Camera_31,66.674969,-114.307091
PERSON,5220372724931202939,1993-05-28T18:39:25,_Camera_20,40.331394,-135.398244
PERSON,5405195789342300017,1986-06-18T01:27:49,_Camera_15,-42.664172,70.18845
PERSON,5423149885582445197,1986-01-12T07:29:29,_Camera_17,-31.420947,36.271061
PERSON,5525421949715910663,2001-09-28T11:49:15,_Camera_24,-64.407795,133.612736
PERSON,561604455246993111,2005-10-27T17:52:25,_Camera_7,38.991455,59.186553
PERSON,570108512295923426,1998-07-13T01:52:59,_Camera_21,39.787559,-140.379141
PERSON,5942023272332926639,1962-05-24T03:34:13,_Camera_5,1.967547,49.305173
PERSON,6142140227037606173,1950-12-01T20:19:17,_Camera_21,39.787559,-140.379141
PERSON,6227065786824529833,1938-11-19T06:55:54,_Camera_42,8.704303,134.719516
PERSON,6264487692430926299,1953-07-10T14:29:32,_Camera_38,-15.906281,-7.659154
PERSON,628621283232104449,1987-02-09T14:20:14,_Camera_12,4.122425,-44.316553
PERSON,631406341810658717,1961-05-21T13:40:46,_Camera_38,-15.906281,-7.659154
PERSON,6335684836173353804,1949-06-24T23:24:48,_Camera_8,-57.389193,75.017573
PERSON,6353384464328059527,1994-08-10T10:47:21,_Camera_13,3.392588,131.009408
PERSON,6376196730250003159,2020-04-11T17:02:34,_Camera_28,71.730761,-58.290246
PERSON,6551881635038631525,1949-02-15T05:19:59,_Camera_15,-42.664172,70.18845
PERSON,6627287640005598996,1984-12-23T15:02:31,_Camera_19,-76.539366,-121.880941
PERSON,6666338407904371979,1972-10-30T19:18:02,_Camera_6,70.834569,176.96735
PERSON,6808056116834300895,1931-06-13T02:27:22,_Camera_36,-12.993477,174.878086
PERSON,6839318193590249664,1949-09-19T05:08:42,_Camera_24,-64.407795,133.612736
PERSON,6987383966571351413,1982-04-01T01:58:35,_Camera_47,17.915955,111.379629
PERSON,712631788407297188,1931-03-19T15:46:09,_Camera_7,38.991455,59.186553
PERSON,7142941759314884259,1998-09-18T03:10:31,_Camera_31,66.674969,-114.307091
PERSON,7210926141352388771,1978-01-24T13:22:23,_Camera_43,45.588239,-82.034685
PERSON,7336208374766282759,2016-05-12T07:24:34,_Camera_6,70.834569,176.96735
PERSON,7530107652603175750,1961-09-13T12:55:49,_Camera_34,-76.036835,88.093573
PERSON,7631406402831711466,1997-07-25T13:27:56,_Camera_28,71.730761,-58.290246
PERSON,7642126307446877622,1962-05-03T15:54:41,_Camera_31,66.674969,-114.307091
PERSON,7950618066225223048,1978-02-14T14:48:12,_Camera_5,1.967547,49.305173
PERSON,8371969411460996596,1960-12-02T14:58:54,_Camera_7,38.991455,59.186553
PERSON,8577277052735484877,1979-09-23T04:48:13,_Camera_16,-40.039633,-151.282119
PERSON,8614139096280875473,1998-09-18T03:10:31,_Camera_45,-36.849172,39.740283
PERSON,86329458279682121,1977-12-02T01:03:02,_Camera_46,-70.676761,-26.562017
PERSON,8784679247694789115,1966-01-15T14:58:32,_Camera_46,-70.676761,-26.562017
PERSON,9046620998864939042,2011-11-24T20:04:21,_Camera_7,38.991455,59.186553
PERSON,9048911914587880100,1997-07-15T13:06:30,_Camera_8,-57.389193,75.017573
PERSON,9181594262443871397,2018-05-21T17:54:01,_Camera_28,71.730761,-58.290246
PERSON,921059820503826440,1941-08-24T01:46:56,_Camera_42,8.704303,134.719516
PERSON,921701348264375437,1995-12-06T09:09:10,_Camera_48,-12.282729,-150.787278

PERSON,1491264515069720082,1930-11-22T02:14:54,_Camera_7,38.991455,59.186553
PERSON,190774122026117782,2012-04-30T00:49:50,_Camera_24,-64.407795,133.612736

PERSON,9046620998864939042,2011-11-24T20:04:21,_Camera_7,38.991455,59.186553
PERSON,9048911914587880100,1997-07-15T13:06:30,_Camera_8,-57.389193,75.017573
PERSON,1012435145270653984,2011-05-27T15:20:04,_Camera_43,45.588239,-82.034685
PERSON,1012435145270653984,1994-01-10T08:05:58,_Camera_42,8.704303,134.719516
PERSON,1012435145270653984,1976-09-30T05:13:47,_Camera_32,41.717836,28.079691
PERSON,1012435145270653984,1959-01-06T21:31:33,_Camera_40,-11.619677,-68.571013
PERSON,1012435145270653984,1951-08-31T21:57:15,_Camera_43,45.588239,-82.034685
PERSON,1012435145270653984,1946-04-24T04:13:25,_Camera_45,-36.849172,39.740283
CAR,00NX08,2008-07-18T12:38:04,_Camera_29,56.088966,-65.741688
CAR,00NX08,2006-09-15T21:59:30,_Camera_6,70.834569,176.96735
CAR,00NX08,2001-04-29T08:39:12,_Camera_9,88.194909,133.790069
CAR,00NX08,1987-10-13T10:44:09,_Camera_7,38.991455,59.186553
CAR,00NX08,1976-06-10T22:32:15,_Camera_13,3.392588,131.009408
CAR,00NX08,1959-02-28T13:41:03,_Camera_5,1.967547,49.305173
CAR,00NX08,1952-11-13T17:03:26,_Camera_15,-42.664172,70.18845
CAR,00NX08,1936-03-06T19:47:27,_Camera_6,70.834569,176.96735
CAR,00NX08,1931-05-16T20:21:28,_Camera_8,-57.389193,75.017573

